kasa/aestransport.py

```diff
@@ -123,17 +147,22 @@ async def send_secure_passthrough(self, request: str):
         }
         status_code, resp_dict = await self.client_post(url, json=passthrough_request)
         # _LOGGER.debug(f"secure_passthrough response is {status_code}: {resp_dict}")
-        if status_code == 200 and resp_dict["error_code"] == 0:
-            response = self._encryption_session.decrypt(  # type: ignore
-                resp_dict["result"]["response"].encode()
+
+        if status_code != 200:
+            raise SmartDeviceException(
+                f"Unable to send message for {self.host}, status code is {status_code}"
```

Comment:
```suggestion
                f"{self.host} responded with an unexpected status code {status_code}"
```
Considering that sending succeeded but the device responded with an error, we should reword this. The above is just a potential suggestion on the wording, so feel free to adapt as you wish.